### PR TITLE
CORE-1147 Add pipelines orchestrate binary

### PR DESCRIPTION
**Pull request by ellisonc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,80 @@
...
+
+The type of event that triggered the workflow. Choose between "push" (for direct pushes to the main branch) or "pr-synched-created" (for pull request synchronization/creation). This input is required.
+
+### `pipelines-version`
+
+The version of the Pipelines CLI to be used (tag, SHA, or branch). This input is required.
+
+### `token`
+
+GitHub Personal Access Token (PAT) required for cloning the Pipelines repository. This input is required.
```
Comment (MoonMoon1919) [inline README.md]:
```suggestion
GitHub Personal Access Token (PAT) required for retrieving the pipelines binary. This input is required.
```

**Review by infraredgirl — DISMISSED** [review]

**Review by oredavids — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,80 @@
...
+
+The most recent commit. This input is required.
+
+### `event-type`
+
+The type of event that triggered the workflow. Choose between "push" (for direct pushes to the main branch) or "pr-synched-created" (for pull request synchronization/creation). This input is required.
+
+### `pipelines-version`
+
+The version of the Pipelines CLI to be used (tag, SHA, or branch). This input is required.
```
Comment (MoonMoon1919) [inline README.md]: Do we only support tags now that we're pulling down releases?

File: action.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        file: 'pipelines_linux_amd64'
+        target: '/tmp/pipelines'
+        token: ${{ inputs.token }}
+
+    - name: Install pipelines
+      shell: bash
+      run: |
+        sudo mv /tmp/pipelines /usr/local/bin/pipelines
+        sudo chmod +x /usr/local/bin/pipelines
+    - name: Run Pipelines Orchestrate
```
Comment (MoonMoon1919) [inline action.yml:52]: lil teeny nit - can you add an additional newline between line 50/51 for readability

File: README.md [hunk 4]
```diff
@@ -1 +1,80 @@
...
\ No newline at end of file
+# Pipelines Orchestrator GitHub Action
+
+This GitHub Action, named "Pipelines Orchestrate," is designed to automate the orchestration of Terragrunt plan/apply/destroy jobs using the Gruntwork Pipelines infrastructure-as-code (IaC) deployment framework. This action helps determine which actions need to take place based on changes to a repository's Terraform or Terragrunt folders.
+
+## Inputs
+
+### `repository-path`
+
+The relative path to the infra-live code repository. This input is required.
```
Comment (MoonMoon1919) [inline README.md]: Should we just `Defaults to "."`?

**Review by infraredgirl — APPROVED** [review]

**Pull request merged**